zombienet_tests/README.md

```diff
@@ -0,0 +1,16 @@
+# Zombie-net tests
+
+_The content of this directory is meant to be used by Parity's private CI/CD infrastructure with private tools. At the moment those tools are still early stage of development and we don't know if / when they will available for public use._
+
+## Contents of this directory
+
+`parachains`
+    At the moment we have only one test for parachains: `/parachains-smoke-test`, that test the parachain registration and the block height.
```

Comment:
This needs an update right?

Reply:
I add a description to this test, is the only one migrated from `simnet`.